### ISSUE TITLE
window border(color) visible, Draw in 'window background'.

### ISSUE THREAD

**Issue by BrunoGysin** [ocornut/imgui#1761]
I followed the code example provided in:
https://github.com/ocornut/imgui/issues/466

```
ImGui::SetNextWindowPos(ImVec2(0,0));
ImGui::SetNextWindowSize(ImGui::GetIO().DisplaySize);
ImGui::Begin("foobar", NULL, ImVec2(0,0), 0.0f, ImGuiWindowFlags_NoTitleBar|ImGuiWindowFlags_NoResize|ImGuiWindowFlags_NoMove|ImGuiWindowFlags_NoScrollbar|ImGuiWindowFlags_NoScrollWithMouse);
```

I've used it since to create an 'invisible' window,
which is the size of the application, which makes it simulate drawing in the background of the application.

but was using an older version of ImGUI, and today I updated.
but the window in which it should be 'invisible' now has a border.

I'm using bg_alpha_override = 0.f 
this was supposed to be working.

image:
https://prnt.sc/j7thki

**Comment by ocornut** [comment]
See https://github.com/ocornut/imgui/issues/1758

**Comment by ocornut** [comment]
Surprisingly this has been coming a few times, I wonder if I should instate a ImGuiWindowFlags_NoBorder that would only affect the WindowBorderSize, but not affect FrameBorderSize or child windows..

**Comment by BrunoGysin** [comment]
I believe that this should be resolved with` bg_alpha_override = 0.0f`

the problem with this solution is that it affects other windows.

**Comment by ocornut** [comment]
You could use `SetNextWindowBgAlpha(0.0f)` but right now it doesn't affect the border (I think it should and i'll change that now).

> the problem with this solution is that it affects other windows.

I don't understand. No it doesn't.
